opentdf-dev.yaml

```diff
@@ -22,6 +22,10 @@ services:
         alg: rsa:2048
         legacy: true
   entityresolution:
+    logger:
+      level: info
+      type: text
```

Comment:
Do we want to customize anything other than level?

Reply:
@elizabethhealy @dmihalcik-virtru 
I think maybe to start we only expose `level`. I don't think we have a use case for different logger types or outputs. 

Can we just simplify this field to `log_level` maybe?